README.md

```diff
@@ -466,6 +480,21 @@ Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 
 
 ### Swift
+- [Swift’i Tanıyalım](https://medium.com/@halilozel1903/swifti-tanıyalım-a5a3a17820ef) - [Halil ÖZEL](https://github.com/halilozel1903)
```

Comment:
Aynı şekilde, yeni eklenenleri sona, var olanları da oldukları yere geri taşırsanız harika olur.

Reply:
Link'in https://medium.com/@halilozel1903/swifti-tan%C4%B1yal%C4%B1m-a5a3a17820ef olarak değişmesi gerek. Travis Türkçe karakterden dolayı 

> 404 https://medium.com/@halilozel1903/swifti-tan 

Olduğunu düşünüp hata döndürüyor. 

Aynı durum birkaç yerde daha mevcut:

> 404 https://medium.com/@halilozel1903/swifti-tan  
>  404 https://medium.com/@halilozel1903/ios-programlamada-admob-kullan  
>  404 https://medium.com/turkishkit/ios-programlamada-navigation-bar-kullan  
>  404 https://medium.com/turkishkit/swift-4-2-ile-uilabel-kullan  
>  404 https://medium.com/turkishkit/swift-ile-nesne-tabanl